canonical-web-and-design/maas.io
# use single contact form

## Done

- use single contact form (/contact-us page)
    - simplify contact-us page form by removing "Country" and "State" fields
    - remove contact-us modal and its dependencies
    
Detailed reasoning for this in the linked issue [#692 ](https://github.com/canonical-web-and-design/maas.io/issues/692#issuecomment-1037099700) approved by design

## QA

1. Go to the contact-us page https://maas-io-695.demos.haus/contact-us and submit the form
2. Make sure everything worked as expected

## Issue / Card

Fixes #692
